test/utils/nargo.js

```diff
@@ -0,0 +1,3 @@
+import { default as path } from "node:path";
+
+export const NARGO_BIN = process.env.NARGO_PATH ? path.resolve(process.env.NARGO_PATH) : "nargo";
```

Comment:
```suggestion
export const NARGO_BIN = process.env.NARGO_BIN ? path.resolve(process.env.NARGO_BIN) : "nargo";
```
Just a nit to keep these aligned if we want to override locally.